src/doc/index.md

```diff
@@ -21,53 +21,81 @@ nav {
 #search-but:hover, #search-input:focus {
     border-color: #55a9ff;
 }
-h2 {
-    font-size: 18px;
+body {
+    font-family: "Source Serif 4", NanumBarunGothic, serif;
+}
+h1, h2, h3, h4, h5, h6 {
+    font-family: "Fira Sans",Arial,NanumBarunGothic,sans-serif;
+}
+h3 {
+    font-size: 1.35rem;
+}
+h4 {
+    font-size: 1.1rem;
 }
 </style>
 
-Welcome to an overview of the documentation provided by the [Rust project].
+<!--
+    Note to editors: make sure external links are indicated with a superscript
+    loop arrow, to help offline users. Code: `<sup>↬</sup>`
```

Comment:
Purely a bikeshed choice - somewhat annoyingly, the "external link" icon we know isn't unicode, so I've seen using `⧉` or any of the [unicode arrows](http://xahlee.info/comp/unicode_arrows.html) instead. Imho the squares and the other arrows look a bit awkwardly big even in superscript so I went with the curly one, but I'll change to whatever you think is best.

<img width="181" alt="image" src="https://user-images.githubusercontent.com/13724985/213585679-dc7b35a8-7632-4d64-b51c-335470fe5672.png">

<img width="190" alt="image" src="https://user-images.githubusercontent.com/13724985/213585472-d9123eb1-0d7f-4c5c-adec-1f5ffcadc31d.png">

<img width="169" alt="image" src="https://user-images.githubusercontent.com/13724985/213585779-83b65093-745f-4e4a-a3fe-98fe68700fb9.png">

Reply:
Hm, yea, it's a bit unfortunate there isn't a readily available standard icon for that.

If I had to choose one, I might go with the last one (the upwards arrow). Often to me these icons often also are used to mean that it will open a new window, which might be a little confusing. I'd be reluctant to choose something that didn't have at least some common usage.

Reply:
Fwiw, the "external link" and "opens in new window" icons are the same. Wikipedia is where I usually see them, and there they mean external link (that opens in the same window)

The reason I made this change at all is because I think a lot of people stumble upon this page when finding the docs offline (myself included when prepping for a flight) and it's nice to see at a glance which pages you can and can't visit. So even if something isn't super common usage, I think it adds real benefit

I actually found one more possibility that might be the winner for me [`The rustc Guide∞`](docs.rs). When I'm back at my computer I'll see how it looks on the actual page, but the hyperlink logo is probably more identifiable than any of the above options. (well, infinity symbol, but close enough)

Reply:
I kind of like normal, not so much superscript

![image](https://user-images.githubusercontent.com/13724985/213849222-bc6de279-8bd4-40b4-9588-6dd3c99c3542.png)

![image](https://user-images.githubusercontent.com/13724985/213849235-f9f0f591-3c5c-49e4-9f16-f8451b050189.png)

Alright, I guess those are the final options. Your pick!

Reply:
I think I'd still lean towards the arrow, as it seems the closest to the more common square with an arrow.

Reply:
I kind of decided that something better than the unicode characters is the actual symbol itself:

<img width="192" alt="image" src="https://user-images.githubusercontent.com/13724985/221259241-58e8f144-68e4-4e78-8d2f-e72f588100f8.png">

I just added a bit of js to add an inline SVG, and also to automatically detect which links are external (so anyone updating this file doesn't have to manually add the tag anymore)

So, problem solved! At least I think